node/network/protocol/src/request_response/mod.rs

```diff
@@ -102,6 +102,19 @@ const STATEMENTS_TIMEOUT: Duration = Duration::from_secs(1);
 /// to have 3 slow nodes connected, to delay transfer for others by `STATEMENTS_TIMEOUT`.
 pub const MAX_PARALLEL_STATEMENT_REQUESTS: u32 = 3;
 
+/// Response size limit for responses of POV like data.
+///
+/// This is larger than `MAX_POV_SIZE` to account for protocol overhead and for additional data in
+/// `CollationFetching` or `AvailableDataFetching` for example. We try to err on larger limits here
+/// as a too large limit only allows an attacker to waste our bandwidth some more, a too low limit
+/// might have more severe effects. See: https://github.com/paritytech/srlabs_findings/issues/101 .
+const POV_RESPONSE_SIZE: u64 = MAX_POV_SIZE as u64 + 10_000;
```

Comment:
10KB seems to be a huge overhead?

Reply:
It is, but it seems to be better to err on the larger side. The limit is only there to limit DoS attacks wasting our bandwidth, from this perspective 10k more are largely irrelevant.